lib/config/presets/index.ts

```diff
@@ -127,6 +128,9 @@ export function parsePreset(input: string): ParsedPreset {
     } else {
       presetName = str.slice(1);
     }
+  } else if (str.includes('//')) {
+    // non-scoped namespace with a subdirectory preset
+    [, packageName, presetPath, presetName] = /(.*?)\/\/(.*)\/(.*)$/.exec(str);
```

Comment:
I think we need to make this regex stricter. it should also be a const in module scope.

Reply:
What kind of changes were you thinking of? I realized root-level files don't parse correctly with the current regex. so I'm thinking of modifying it to this: `/(.*?)\/\/(?:(.*)\/)?(.*)$/` ([RegExr link](https://regexr.com/5mqcc)).